<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="le_komentarz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_zdjecie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_timer">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_poprzedni">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Poprzedni</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_nastepny">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Nastepny</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_pokaz">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaz slajd√≥w</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
